### DISCUSSION TITLE
geospatial tracking

### DISCUSSION THREAD

**Discussion by Giantbean** [google/model-viewer#4855] [Ideas]
Getting Google Geospatial creator and model-viewer together would be a match made in heaven. Just using this tool for a day, I love how solid the tracking is in model viewer. I just wish I could also put in a LAT: and LONG: to set up models around a site for AR walking tours with this web interface instead of needing Adobe Aero or Unity 3D.

**Comment by elalish** [comment]
We have talked about this from time to time - the main trouble is getting the geospatial tracking data. As I recall those APIs weren't really set up with the web in mind. If you have ideas, PRs welcome!

**Reply by Giantbean** [reply]
Could it use HTML5 Geolocation? It seems the Geospatial Creator team could still help make that shine.

**Reply by elalish** [reply]
I have no idea, but if you want to link to a prototype, I'd love to see what's possible.
